### PR TITLE
Edited readme

### PR DESCRIPTION
**Pull request by bukunmialuko**
Implemented enable and disable user.

### CODE REVIEW

File: pkg/user/service/userservice.go [hunk 1]
```diff
@@ -20,6 +20,45 @@ type service struct {
...
+	if user.IsEnabled == false {
+		user.IsEnabled = true
+		_, err := s.repository.Update(*user)
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+
+	return errors.New("user is already enabled")
```
Comment (danieloluwadare) [inline pkg/user/service/userservice.go]: @bukunmialuko .. if the user is already enabled don't return an error...return nil

File: pkg/user/service/userservice.go [hunk 2]
```diff
@@ -20,6 +20,45 @@ type service struct {
...
+	if user.IsEnabled == true {
+		user.IsEnabled = false
+		_, err := s.repository.Update(*user)
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+
+	return errors.New("user is already disabled")
```
Comment (danieloluwadare) [inline pkg/user/service/userservice.go]: This same thing goes here if user is already reviewed no need to return error.

**Review by danieloluwadare — COMMENTED** [review]
Kindly check the review  and make the necessary change

File: pkg/user/service/userservice.go (hunk 2 above)
Comment (bukunmialuko) [reply pkg/user/service/userservice.go]: Done

**Review by danieloluwadare — COMMENTED** [review]
Kindly test your route..

**Comment by bukunmialuko** [comment]
> Routes tested. ready to go!

**Pull request merged**